scripts/example_configs/femnist/avg/fts.yaml

```diff
@@ -0,0 +1,63 @@
+use_gpu: True
+device: 2
+outdir: exp/fts1
+early_stop:
+  patience: 100
+seed: 12345
+federate:
+  client_num: 5
+  mode: standalone
+  sample_client_rate: 1.0
+  share_local_model: True
+  online_aggr: True
+  use_diff: True
+data:
+  root: data/
+  type: femnist
+  splits: [0.6,0.2,0.2]
+  batch_size: 16
+  subsample: 0.05
+  transform: [['ToTensor'], ['Normalize', {'mean': [0.1307], 'std': [0.3081]}]]
+  num_workers: 0
+model:
+  type: convnet2
+  hidden: 2048
+  out_channels: 62
+  dropout: 0.5
+train:
+  batch_or_epoch: epoch
+  local_update_steps: 1
+  optimizer:
+    lr: 0.01
+    weight_decay: 0.0
+grad:
+  grad_clip: 5.0
+criterion:
+  type: CrossEntropyLoss
+trainer:
+  type: cvtrainer
+eval:
+  freq: 1
+  metrics: ['acc', 'correct', 'f1']
+  split: ['test', 'val', 'train']
+hpo:
+  scheduler: 'fts'
```

Comment:
Sorry for my mis-understanding of this work at that time. For now, is it necessary to have a FTS scheduler as the entry? If it is the case, where can I find the implementation (it seems that nothing in algo.py)?